src/dsp_tools/commands/excel2json/lists.py

```diff
@@ -71,6 +71,45 @@ def expand_lists_from_excel(
     return new_lists
 
 
+def excel2lists(
+    excelfolder: str,
+    path_to_output_file: Optional[str] = None,
+    verbose: bool = False,
+) -> tuple[list[dict[str, Any]], bool]:
+    """
+    Converts lists described in Excel files into a "lists" section that can be inserted into a JSON project file.
+
+    Args:
+        excelfolder: path to the folder containing the Excel file(s)
+        path_to_output_file: if provided, the output is written into this JSON file
+        verbose: verbose switch
+
+    Raises:
+        UserError: if something went wrong
+        BaseError: if something went wrong
+
+    Returns:
+        a tuple consisting of the "lists" section as Python list, and the success status (True if everything went well)
+    """
+    # read the data
+    excel_file_paths = _extract_excel_file_paths(excelfolder)
+    if verbose:
+        print("The following Excel files will be processed:")
+        print(*(f" - {filename}" for filename in excel_file_paths), sep="\n")
+
+    # construct the "lists" section
+    finished_lists = _make_json_lists_from_excel(excel_file_paths, verbose=verbose)
```

Comment:
Now, the modifier "finished" is pointless

Reply:
What do you mean?

Reply:
In the old code, it was perhaps important to distinguish between "lists" and "finished lists". But if I read the code now, I think these are just the "lists". There is no need to stress that these lists are finished.